tests/check_framework/models.py

```diff
@@ -6,10 +6,14 @@ class SimpleModel(models.Model):
     field = models.IntegerField()
     manager = models.manager.Manager()
 
+class CheckModel(models.Model):
+    check_ran = models.BooleanField(default=False)
+
 
 @register('tests')
 def my_check(app_configs, **kwargs):
     my_check.did_run = True
```

Comment:
It shouldn't be necessary to have two ways of checking this, the model instance should be enough?

Reply:
Agreed, I forgot to remove this